keps/sig-release/3027-slsa-compliance/README.md

```diff
@@ -0,0 +1,894 @@
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+Release Kubernetes in a zero-trust environment.
+
+Kubernetes releases represent key links in many software supply chains, not
+just for the project itself but also for consumers that derive, repackage and
+distribute our artifacts downstream. The project releases end-user artifacts
+like binaries and container images, but also source code that is actively reused
+further down the distribution stream.
+
+All current work done by SIG Release on the Kubernetes supply chain centers
+around three focus areas: Artifact Consumption, Introspection, and Security.
+These areas are explained in more detail in our [_Roadmap for 2021 and Beyond_ 
+document](https://github.com/kubernetes/sig-release/blob/master/roadmap.md).
+
+As the world hardens its software distribution methods, the Kubernetes project needs
+to do its part to achieve a secure supply chain from end to end: from the
+top base images to the final artifacts downloaded by end users. This proposal
+provides a path to achieve increasing levels of security, integrity, and availability
+in our releases by engineering new features to our processes. The objective is
+to achieve the highest possible compliance with the
+[SLSA framework](https://slsa.dev/) (Supply-chain Levels for Software Artifacts).
+
+We consider SLSA compliance to be an effort in line with the three objectives outlined
+in our roadmap: Artifacts can be consumed easier and with more trust.
+Improvements to code and process will secure the supply chain and each release
+will produce software bills of materials, provenance attestations and signatures
+which will yield much better introspection to the journey from code to binary.
+
+The SLSA framework is a project under the [OpenSSF](https://openssf.org/)'s [Digital Identity 
+Attestation Working Group](https://github.com/ossf/wg-digital-identity-attestation).
+The framework defines numbered levels of compliance that harden software supply
+chains by recommending concrete steps to address, each of increasing technical
+complexity, number of stakeholders, and process adjuments necessary to reach them. 
+
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+The main goal of this KEP is to provide a plan to achieve the highest possible
+SLSA compliance in the Kubernetes release process. While providing a roadmap
```

Comment:
As already mentioned in the non-goals below, we do not provide a plan. An incomplete plan maybe, but I'd like to rephrase it more to state that the main goal of this kep is the highest SLSA compliance level for k/k, for example:
```suggestion
The main goal of this KEP is to define the highest possible
SLSA compliance as final objective in enhancing the Kubernetes release process. While providing a roadmap
```